cocktails.web/.env.loc

```
@@ -1,15 +1,15 @@
+VITE_AUTH0_DOMAIN=cezzis.us.auth0.com
+VITE_AUTH0_CLIENT_ID=HZXyiZxjHgkQqeb6UJxmikCGYpSz5iPb
+VITE_AUTH0_REDIRECT_URI=https://localhost:4001/
+VITE_AUTH0_COCKTAILS_API_AUDIENCE=https://cezzis-cocktails-api
+
 VITE_COCKTAILS_API_URL=https://localhost:7176
 VITE_COCKTAILS_IMAGE_URL=https://localhost:7176/api/v1/images
 VITE_RECAPTCHA_SITE_KEY=6Le3QR0qAAAAAIAUOYQO7P6KoFCVdZuAR1FwvUnG
-VITE_COCKTAILS_APIM_SUBSCRIPTION_KEY=383hudiudhUJK984jdus7HDY
-VITE_LOGIN_SUBDOMAIN=login
\ No newline at end of file
+VITE_COCKTAILS_APIM_SUBSCRIPTION_KEY=383hudiudhUJK984jdus7HDY
```

dotenv-linter (3.3.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)